templates/taxi/car_list.html

```diff
@@ -7,6 +8,12 @@ <h1>
       Create
     </a>
   </h1>
+  
+  <form method="get" action="" class="form-inline">
+    {{ search_form|crispy }}
+    <input type="submit" value="Submit" class="btn-btn-primary">
```

Comment:
There is a typo in the class attribute value. It should be 'btn btn-primary' instead of 'btn-btn-primary'.